lib/routes/account.js

```diff
@@ -213,7 +213,7 @@ module.exports = function (
               emailCode: account.emailCode,
               emailVerified: account.emailVerified,
               verifierSetAt: account.verifierSetAt,
-              createdAt: optionallyOverrideCreatedAt(),
+              createdAt: parseInt(query._createdAt),
```

Comment:
Thanks @vladikoff, this was a great catch and now I understand why the code was the way it was before (initially setting the token's `createdAt` to the account's `createdAt`, then changing it to `Date.now()` later on in `Token.createNewToken`). Will fix it and also add some test coverage for `accountCreatedAt`.